.graphqlconfig

```diff
@@ -0,0 +1,14 @@
+{
+  "schemaPath": "schema.graphql",
+  "extensions": {
+    "endpoints": {
+      "Default Introspection Endpoint(local lambda server)": {
+        "url": "http://localhost:8000/___graphql",
```

Comment:
This will only be used by GraphQL tooling, like the IntelliJ JS GraphQL plugin, 
which allows validation and Syntax Completion based on the configured schema.
[https://plugins.jetbrains.com/plugin/8097-js-graphql/]

So this file is an optional, extra file (not part of gatsby itself)

It will be only available when gatsby is started in dev mode.
Needs to be documented somehow in the [READ.me], later